pkg/ovs/ovsctl/appctl.go

```diff
@@ -166,3 +191,39 @@ func (c *ovsCtlClient) RunAppctlCmd(cmd string, needsBridge bool, args ...string
 	}
 	return out, nil
 }
+
+func (c *ovsCtlClient) ShowDPFeatures() (map[DPFeature]bool, error) {
```

Comment:
Even it executes "show-dp-features", we might call this func GetDPFeatures()?

Reply:
Done